app/javascript/packs/stylesheets/enrollment.scss

```diff
@@ -18,3 +18,22 @@
 .first-positive-lab-result-field-name {
   font-weight: 600;
 }
+
+.common-exposure-cohort-table-wrapper {
```

Comment:
this name is way too close to the one in `patient.scss` and it's confusing that they're in different files (I do get it though).

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/9224bca33be59de6ff1b6a3827c57c21f0a27a87#diff-95e955f88f7825225541c2bbdd1cd2c305f473e799de1ba90a97229cb323dcd3R22